Lib/pathlib.py

```diff
@@ -686,7 +686,9 @@ def _parse_args(cls, args):
         return cls._flavour.parse_parts(parts)
 
     @classmethod
-    def _from_parts(cls, args, init=True):
+    def _from_parts(cls, args, kwargs=None, *, init=True):
```

Comment:
It looks like `kwargs` is not needed as it's not used anywhere?

Reply:
It has been removed on main, you have a merge conflict now.